### PR TITLE
chore: Changes for alpha v5 release

### LINKED ISSUE #1177: Refactor package name and update app version
### Chore

**Parent Issue**
#809 

**Description**

1. Refactor package name from `org.fossasia.openevent` to `com.eventyay.organizer`.
2. Update app version.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #1177

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 

1.  Refactored application id from `org.fossasia.eventyay` to `com.eventyay.organizer`.
2.  Updated app version.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
No need to change java package. Just change applicationId in build.gradle

File: app/build.gradle [hunk 1]
```diff
@@ -19,11 +19,11 @@ android {
...
     buildToolsVersion versions.buildTools
     defaultConfig {
-        applicationId "org.fossasia.eventyay"
+        applicationId "com.eventyay.organizer"
         minSdkVersion versions.minSdk
         targetSdkVersion versions.targetSdk
-        versionCode 4
-        versionName "1.0.3alpha"
+        versionCode 5
+        versionName "1.0.0beta"
```
Comment (iamareebjamal) [inline app/build.gradle]: Should be 1.0.4alpha

Comment (Masquerade0097) [reply app/build.gradle]: Ok, I've changed it to alpha but Mario told in the gitter chat that we need to release beta version now. Aren't we releasing beta ?

Comment (iamareebjamal) [reply app/build.gradle]: We'll release in the beta channel on Play Store, the app itself is in alpha phase

Each release of app on Play Store goes through alpha -> beta -> release

Even Whatsapp which is not in alpha or beta, Play Store channels are unrelated to app state

**Comment by Masquerade0097** [comment]
@sridharjajoo The version code for this release is 5, therefore, v5 is used in PR title. Don't confuse it with version name.

**Comment by iamareebjamal** [comment]
Follow up issue would be to change the package name in upload apk script

**Pull request merged**